package/MDAnalysis/analysis/pcz.py

```diff
@@ -0,0 +1,481 @@
+data. For examples of the application of PCA to MD analysis see Wlodek et al.
+[Wlodek1997]_ and Sherer et al. [Sherer1999]_.
+
+Much of the code has been adapted from that in the `pyPcazip package`_.
+
+.. Rubric:: References
+
+..[Wlodek1997] Wlodek ST, Clark TW, Scott LR, McCammon JA (1997)
+               J Am Chem Soc 119, 9513-9522.
+
+..[Sherer1999] Sherer EC, Harris SA, Soliva R, Orozco M, Laughton CA (1999)
+               J Am Chem Soc 121, 5981-5991.
+
+.._`pyPcazip package`: https://bitbucket.org/ramonbsc/pypcazip
+
+
+
+.. examples
+
+Examples
+--------
+
+    import MDAnalysis as mda
+    from MDAnalysis.analysis.pca import Pca
+
+    u = mda.Universe(pdb, trajectory)
+    p = Pca(u, atom_selection='name CA')
+    # print the eigenvalues:
+    print p.evals
+    # print the projection of the trajectory along the second eigenvector:
+    print p.proj(1)  # note zero-based indexing!
+    # print the scores for trajectory frame 7:
+    print p.scores(7)
+    # print the scores for a different coordinate set:
+    u2 = mda.Universe(pdb, trajectory2)
+    x = u.atomSelection('name CA').ts._pos
+    print p.transform(x)
+    # generate a coordinate set from a scores vector defining
+    # a point in (2D) PC space:
+    s = [5.0, 3.0]
+    x = p.transform(s, inverse=True)
+
+
+Helper functions
+----------------
+The following are convenience classes and functions primarily for internal use.
+
+.. autoclass:: Fasu
+.. autofunction:: lsqfit
+.. autofunction:: rmsd
+
+Classes, methods, and functions
+-------------------------------
+
+.. autoclass:: Pca
+   :members:
+
+   .. attribute:: natoms
+
+      int, number of atoms selected for PC analysis
+
+   .. attribute:: numframes
+
+      int, number of trajectory frames selected for PC analysis
+
+   .. attribute:: avg
+
+      :class:`numpy.ndarray` containing the coordinates of the average structure
+
+   .. attribute:: nvecs
+
+      int, the number of eigenvectors/eigenvalues calculated
+
+   .. attribute:: captured_variance
+
+      float, the fraction (between 0.0 and 1.0) of the total variance captured 
+      by the *nvecs* eigenvectors
+
+   .. attribute:: evals
+
+      :class:`numpy.ndarray` containing the eigenvalues (largest first)
+
+   .. attribute:: evecs
+
+      :class:`numpy.ndarray` containing the eigenvectors
+
+"""
+import MDAnalysis as mda
+import numpy as np
+
+from MDAnalysis.analysis.align import rotation_matrix
+from scipy.linalg import eigh
+
+
+def lsqfit(mobile, reference, weights = None):
+    """
+    Utility function to fit the coordinates in the [n, 3] numpy array
+    *mobile* to those in the [n, 3] array *reference*, with optional mass
+    weighting.
+
+    :Arguments:
+      *mobile*
+        :class:`numpy.ndarray` containing the coordinates to be fitted
+
+      *reference*
+        :class:`numpy.ndarray` containing the target coordinates
+
+      *weights*
+        :class: numpy.ndarray`  (optional) for mass-weighted fitting
+
+    :Returns:
```

Comment:
It would still be nice if you change the doc-strings. We have recently decided on a [style guide](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide) and the numpy-docs are part of that. We haven't switched all the old-code at once but are fixing all the issues with it as we go along. To help with this new code should follow the [style guide](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide).